NickvisionMoney.WinUI/Views/PreferencesPage.xaml.cs

```diff
@@ -185,10 +185,12 @@ private string RemoveUrlsFromCredits(string s)
     /// <param name="e">RoutedEventArgs</param>
     private async void Help(object sender, RoutedEventArgs e)
     {
-        var lang = CultureInfo.CurrentCulture.ToString() switch {
-            "ru-RU" => "ru",
-            _ => "C"
-        };
+        var lang = "C";
+        var availableTranslations = new string[2] {"es", "ru"};
+        if (availableTranslations.Contains(CultureInfo.CurrentCulture.TwoLetterISOLanguageName.ToString()))
```

Comment:
`CultureInfo.CurrentCulture.TwoLetterISOLanguageName` is already a string, therefore the extra `ToString()` is not needed.

https://learn.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.twoletterisolanguagename?view=net-7.0